README.md

```diff
@@ -7,10 +7,10 @@
 
 Stagehand provides a modern, type-safe API for building animations on iOS. Stagehand is designed around a set of core ideas:
 
-* Composition of Structures
-* Separation of Construction and Execution
-* Compiler Safety
-* Testability
+* **Composition of Structures** - Stagehand makes it easy to build complex, multi-part animations that are built from small, reusable pieces that are easier to reason about.
+* **Separation of Construction and Execution** - Stagehand provides separate mechanisms for the construction and execution, which increases the flexibility of animations and makes concepts like queuing a series of animations work straight out of the box.
```

Comment:
`flexibility of animations` sounds more like flexibility in curve control and transformation, but it is actually about the flexibility of timing and order of execution right?

Reply:
Yeah, good call out. Maybe "flexibility of animation code"? Or maybe "flexibility" is the actually wrong word here. Some examples of what I'm talking about here:

* You can construct an animation and wait until another event happens to execute it, or even add it to a queue of animations.
* You can abstract away complex construction behind other structures (e.g. animation factories) and keep your execution simple. This ties in closely with the composition of structures.
* You could have the construction and execution of your animations live in different parts of your architecture. Since animations are simply value types, you can pass them around with your other data until you're ready to execute them.
* In theory, if you have a super complex animation that's expensive to create, you could even construct it on a background thread and then execute it on the main thread when you're ready to go.

I don't think there's really one core advantage to separating the construction and execution. Stagehand treats them as conceptually different tasks, which gives you more flexibility in how your structure your code.